src/main/java/interfaces/IServerSocket.java

```diff
@@ -0,0 +1,8 @@
+package interfaces;
```

Comment:
**suggestion** Put your interfaces in the same package that will depend on them.

The Dependency Inversion Principle offers some guidance here.